examples/dockerdev/docker-compose.yml

```
@@ -24,7 +24,7 @@ x-backend: &backend
   stdin_open: true
   tty: true
   volumes:
-    - .:/app:cached
+    - .:/app:cached,z
```

Let's add it as an example (not the default) with the link to the docs:

```suggestion
    - .:/app:cached
    # or for selinux: https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label
    # - .:/app:cached,z
```

I think, "Use extreme caution with these options" should be considered by the end users.